### PR TITLE
fixed staticcheck errors in folder cmd

### PR DESCRIPTION
**Pull request by elitecodegroovy**
### What problem does this PR solve? 
staticcheck tool reported the issues by executing the following commands:
```
GO111MODULE=on staticcheck ./...
```
Expected no output.  [The same type issue that was reported](https://github.com/pingcap/tidb/issues/13456).

### Check List 

Integration test

Code changes

Changed the test case.

Side effects

No.

Related changes

No.

### CODE REVIEW

File: server/schedulers/balance_test.go [hunk 1]
```diff
@@ -1211,7 +1211,7 @@ func (s *testScatterRangeLeaderSuite) TestConcurrencyUpdateConfig(c *C) {
 		for {
 			select {
 			case <-ch:
-				break
+				//break
```
Comment (disksing) [inline server/schedulers/balance_test.go]: change to `return`?

File: server/config/config_test.go [hunk 2]
```diff
@@ -209,7 +209,7 @@ enable-remove-extra-replica = false
 	c.Assert(cfg.Schedule.EnableRemoveDownReplica, IsFalse)
 	c.Assert(cfg.Schedule.EnableMakeUpReplica, IsFalse)
 	c.Assert(cfg.Schedule.EnableRemoveExtraReplica, IsFalse)
-	b, err := json.Marshal(cfg)
+	b, err := json.Marshal(cfg.Schedule)
```
Comment (disksing) [inline server/config/config_test.go]: Why need to update it?

Comment (elitecodegroovy) [reply server/config/config_test.go]: Please look the issue reported by the commands: 
```
GO111MODULE=on staticcheck ./...
```

Comment (disksing) [reply server/config/config_test.go]: I think it's false positive report.

Comment (elitecodegroovy) [reply server/config/config_test.go]: > I think it's false positive report.

The idea is from this issue reported [https://github.com/pingcap/tidb/issues/13456](url)

Comment (elitecodegroovy) [reply server/config/config_test.go]: > I think it's false positive report.

siddontang said that CI will use the tool to validate the Go code.

Comment (elitecodegroovy) [reply server/config/config_test.go]: > I think it's false positive report.

No break is needed in the default case.

File: server/schedulers/balance_test.go (hunk 1 above)
Comment (elitecodegroovy) [reply server/schedulers/balance_test.go]: > change to `return`?

No break is needed in the default case.

Comment (disksing) [reply server/schedulers/balance_test.go]: I think it should be `return` here

File: server/config/config_test.go (hunk 2 above)
Comment (disksing) [reply server/config/config_test.go]: I don't care what he said. There is nothing wrong with the code. The tool has bug to report it.

File: server/schedulers/balance_test.go (hunk 1 above)
Comment (lhy1024) [reply server/schedulers/balance_test.go]: why comment out it? we need to stop it here.

File: server/config/config_test.go (hunk 2 above)
Comment (elitecodegroovy) [reply server/config/config_test.go]: > I don't care what he said. There is nothing wrong with the code. The tool has bug to report it.

OK! Please look the the comments of the func `json.Marshal`. It described as the following:
```
// Channel, complex, and function values cannot be encoded in JSON.
// Attempting to encode such a value causes Marshal to return
// an UnsupportedTypeError.
```

Now, `cfg` is a function.

File: server/schedulers/balance_test.go (hunk 1 above)
Comment (elitecodegroovy) [reply server/schedulers/balance_test.go]: > why comment out it? we need to stop it here.

ineffective break statement. Did you mean to break out of the outer loop? (SA4011)

Comment (elitecodegroovy) [reply server/schedulers/balance_test.go]: > why comment out it? we need to stop it here.



> change to `return`?

Change to `return` is a good choice.

File: server/config/config_test.go (hunk 2 above)
Comment (disksing) [reply server/config/config_test.go]: why is `cfg` a function?

Comment (elitecodegroovy) [reply server/config/config_test.go]: > why is `cfg` a function?

Maybe the tool has a bug , but what I changed doesn't affect the execution. Ignore the case if you think it is not problem.

Comment (disksing) [reply server/config/config_test.go]: I think it's not a problem. Please revert this line.

File: server/schedulers/balance_test.go (hunk 1 above)
Comment (disksing) [reply server/schedulers/balance_test.go]: Please update it.

Comment (elitecodegroovy) [reply server/schedulers/balance_test.go]: > Please update it.

Done.

File: server/config/config_test.go (hunk 2 above)
Comment (elitecodegroovy) [reply server/config/config_test.go]: > I think it's not a problem. Please revert this line.

Done.

**Pull request merged**